creator-node/src/monitors/MonitoringQueue.js

```diff
@@ -0,0 +1,111 @@
+const Bull = require('bull')
+const redis = require('../redis')
+const config = require('../config')
+const { logger } = require('../logging')
+const { getDatabaseSize } = require('./database')
+
+const MONITORING_REDIS_PREFIX = 'monitoring'
+const PROCESS_NAMES = Object.freeze({
+  monitor: 'monitor'
+})
+
+/**
+ * List of all monitors to run, containing:
+ *  @param {string} name A unique name (for caching in redis)
+ *  @param {number} ttl TTL in seconds for how long a cached value is good for
+ *  @param {function} func The actual work to compute a value. The return value is what is cached.
```

Comment:
nit - I don't think function is a valid param type https://jsdoc.app/tags-param.html#callback-functions

Reply:
```
The parameter type can be a built-in JavaScript type, such as string or Object, or a JSDoc namepath to another symbol in your code. If you have written documentation for the symbol at that namepath, JSDoc will automatically link to the documentation for that symbol. You can also use a type expression to indicate, for example, that a parameter is not nullable or can accept any type; see the @type tag documentation for details.
```

<img width="170" alt="Screen Shot 2021-02-02 at 11 13 30 AM" src="https://user-images.githubusercontent.com/2731362/106650480-ceb53e80-6547-11eb-84e1-f87c78eb8e1d.png">

Reply:
oooooooooooooo interestinggggggg okkkkkkk ty